.devops/azure-templates/setup-env.yml

```diff
@@ -44,11 +44,11 @@ parameters:
     type: string
     default: ''
 
-  - name: 'ecommerce_host'
+  - name: 'api_ecommerce_basepath'
     type: string
     default: ''
 
-  - name: 'api_ecommerce_basepath'
```

Comment:
why removing this variable? it's used in other system properties such as `CHECKOUT_API_ECOMMERCE_BASEPATH`